app/scripts/controllers/transactions/index.js

```diff
@@ -1353,6 +1357,15 @@ export default class TransactionController extends EventEmitter {
     // So that we do not increment nonce + resubmit something
     // that is already being incremented & signed.
     const txMeta = this.txStateManager.getTransaction(txId);
+
+    ///: BEGIN:ONLY_INCLUDE_IN(mmi)
```

Comment:
MMI does not broadcast transactions, as that is the responsibility of the custodian